### ISSUE TITLE
Container preview renders parts of the GUI incorrectly when a resource pack is enabled

### ISSUE THREAD

**Issue by Arcensoth** [maruohon/minihud#21]
It looks like the bottom-right edges of the GUI are being ripped from the wrong edges of the container texture:

![image](https://user-images.githubusercontent.com/1885643/66420452-e0c6b180-e9d3-11e9-87d1-0487e2485569.png)

Note that it is possible to change the shullker box GUI texture in vanilla. (Whereas OptiFine allows you change it based on the *colour* of the box.)

A similar issue can be seen with the chest preview, which incorrectly renders the shulker box GUI:

![image](https://user-images.githubusercontent.com/1885643/66420406-cd1b4b00-e9d3-11e9-884d-44b6889a289e.png)

Resource packs:
- https://www.curseforge.com/minecraft/texture-packs/wood-and-stone-gui

Relevant mods:
- fabric-api-0.4.0+build.240-1.14
- malilib-fabric-1.14.4-0.10.0-dev.20
- minihud-fabric-1.14.4-0.19.0-dev.20191007.003640

Fabric loader:
- fabric-loader-0.6.3+build.167-1.14.4

**Comment by maruohon** [comment]
Yes these background renderers can be improved, and in the 1.12 branch there is a new rendering util method for 9-splicing, which should probably also be used for this. It will probably be difficult to splice these together in a way that would work properly for most resource packs, as they might have different styling for the texture edges, which would break with different resource packs with different splicing implementations. If I remember right, the single chest doesn't have a separate texture in vanilla, I think it probably uses the double chest texture(?) But I need to check that.
